javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-playlist-app/index.html

```diff
@@ -2,124 +2,80 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>freeCodeCamp Music Playlist Project</title>
-    <link rel="stylesheet" href="./styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&family=Roboto+Mono&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Winamp-style Music Player</title>
   </head>
-
   <body>
-    <header class="header">
-      <h1>Music Playlist Project</h1>
-    </header>
-
-    <main class="container">
-      <div class="music-player">
-        <div class="flex-container">
-          <div class="current-song-container">
-            <img class="img-styles" alt="freeCodeCamp logo" src="./mic.jpg" />
-            <div class="text-container"></div>
+    <div class="container">
+      <div class="player">
+        <div class="player__bar">
+          <div>
+            <img src="assets/decor-lines.svg" alt="decoration lines left" />
```

Comment:
```suggestion
            <img src="assets/decor-lines.svg" alt="" />
```

These are purely decorative images and should be hidden from screen reader users by using empty `alt` text.